src/client/css/partials/breaches.css

```diff
@@ -186,7 +186,7 @@
      so that everything remains vertically-aligned, and the badge still
      disappears when getting flex-wrapped on small screens.
   */
-  font-size: 11px;
+  font-size: 0.75rem;
```

Comment:
It's an ugly number, but should this be `0.6875rem` to "equal" 11px?  Alternatively, I could ask UX if they might stick to even-number font-sizes...